src/App.vue

```diff
@@ -1189,6 +1192,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
   private async loadKeycloakRoles (): Promise<string[]> {
     const jwt = getJWT()
     const keycloakRoles = (jwt.roles || []) as Array<string>
+    console.log('*** keycloak roles =', keycloakRoles)
```

Comment:
I'll remove this (and below) eventually but I'm still working through the roles or permissions we're going to use for the different user types:
- clients (regular users)
- BC Registry Staff
- Service BC Staff
- BCOL Help Desk
- Maximus Contact Centre

Reply:
Obsolete comment -- I removed KC roles in favour of using Authorizations Roles (from Auth API).

Also, see code for the different "account types".